src/config.ts

```diff
@@ -0,0 +1,11 @@
+'use strict'
```

Comment:
@slimsag @felixfbecker is this useful/necessary?

Reply:
https://stackoverflow.com/questions/31391760/use-strict-needed-in-a-typescript-file

We should enable the always strict TS compiler flag I guess

Reply:
https://github.com/sourcegraph/sourcegraph-vscode/pull/17